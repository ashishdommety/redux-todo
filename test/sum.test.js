const sum = require('../src/sum.js');
// import sum from '../src/sum.js';

it('adds two numbers',() => {
  expect(sum(1,2)).toMatchSnapshot();
});
