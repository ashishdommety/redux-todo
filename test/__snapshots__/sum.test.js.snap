// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds two numbers 1`] = `3`;
